skpro/distributions/halfnormal.py

```diff
@@ -0,0 +1,210 @@
+# copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
+"""Half-Normal probability distribution."""
+
+__author__ = ["SaiRevanth25"]
+
+import numpy as np
+import pandas as pd
+from scipy.special import erf, erfinv
+
+from skpro.distributions.base import BaseDistribution
+
+
+class HalfNormal(BaseDistribution):
+    r"""Half-Normal distribution (skpro native).
+
+    This distribution is univariate, without correlation between dimensions
+    for the array-valued case.
+
+    The half-normal distribution is parametrized by the standard deviation
```

Comment:
I think you need to say something about the domain, or that pdf is zero for negative.

Also, please clarify whether this is truncated or clipped, as it might not be clear to the reader: that is, is there a mass on zero?

Reply:
Updated!